cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -842,11 +843,14 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
   toObject _verb _ =
     mkObject [ "kind" .= String "TraceLocalTxSubmissionServerEvent" ]
 
-instance ProtocolLedgerView blk => ToObject (TraceForgeEvent blk tx) where
-  toObject _verb (TraceAdoptedBlock slotNo _blk _txs) =
+instance (HasTxId tx, ProtocolLedgerView blk, Condense (HeaderHash blk), Show (TxId tx))
+           => ToObject (TraceForgeEvent blk tx) where
+  toObject _verb (TraceAdoptedBlock slotNo blk txs) =
```

Comment:
we should probably print the list of transaction ids only if _verb == 'MaximumVerbosity'
can potentially be huge!

Reply:
@CodiePP changes made